corehq/apps/accounting/async_handlers.py

```diff
@@ -234,6 +234,22 @@ def domain_response(self):
         return [(d, d) for d in domain_names]
 
 
+class Select2CustomerInvoiceTriggerHandler(BaseSelect2AsyncHandler):
+    slug = 'select2_billing'
+    allowed_actions = [
+        'customer_account',
+    ]
+
+    @property
+    def customer_account_response(self):
+        customer_accounts = []
+        for account in BillingAccount.objects.all():
+            if account.is_customer_billing_account:
```

Comment:
Filtering in the query (instead of in python) means that Postgres will filter, which avoids fetching the entire table.  This is recommended for performance.